### PR TITLE
Feature/130 download button

### PR DESCRIPTION
**Pull request by maxdiebold-erg**
## Related Issues:
* [EQ-130](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=865&view=detail&selectedIssue=EQ-130)

## Main Changes:
* Added functionality to the _Download_ button in the client interface.
* Added a modal that displays the current query's row count before executing the query and downloading data.
    * The OOTB **USWDS** modal component did not work as expected, so a custom `Modal` component was created, based heavily on the homonymous component from the [react-uswds](https://github.com/trussworks/react-uswds) repository. This component has more features than we need: they can be left in or further trimmed down.
* Added a missing search icon to the static **HTML** pages.
* Removed the `uswds.js` import from `app.tsx`, since it was interfering with the EPA template.

## Steps To Test:
1. Navigate to http://localhost:3000/attains#dataProfile=assessmentUnits&format=xlsx&assessmentUnitId=AL03130002-0804-400.
2. Click the _Download_ button, then click _Continue_ in the modal that appears.
3. Verify that the file download process proceeds as expected, and that the file contents are correct.
4. Build a query for a profile that is not yet connected, e.g. http://localhost:3000/attains#dataProfile=assessments&format=xlsx&assessmentUnitId=AL03130002-0804-400.
5. Click the _Download_ button, and confirm that an error message is displayed in a modal.
6. Repeat as needed with different profiles, filters, and file formats.

## TODO:
- [ ] We'll need to establish a row count cutoff before sending users to the national downloads, as well as the UI behavior once that cutoff is exceeded.

### CODE REVIEW

**Comment by cschwinderg** [comment]
I'm still reviewing this PR, but one thing I noticed is the `eqDataUrl` was using the `window.location` only when running locally. 

This would cause it to break on the dev site because we have `https://api.epa.gov/expert-query/attains` for the `eqDataApi` variable in `services.json`. That was my bad, back when I created the #27 PR, I forgot to remove that test url from the `eqDataApi` variable. 

I went ahead and pushed a commit to revert that stuff back and empty the `eqDataApi` variable. Sorry, about the confusion around that.

**Review by cschwinderg — CHANGES_REQUESTED** [review]
Looks good and works well! Everything works perfectly as is. 

One suggestion I have (more info below) is to switch from using `focus-trap-react` to using `@reach/dialog`. This will likely simplify our code-base and make this code a little more consistent with how we do modals in some of our other apps (clean school bus, HMW, TOTS, etc.).

File: app/client/package.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
     "@reach/portal": "0.18.0",
     "@reach/tooltip": "0.18.0",
     "bootstrap": "5.2.2",
+    "focus-trap-react": "10.0.2",
```
[the remark this answers is not in the document]
Comment (maxdiebold-erg) [reply app/client/package.json:40]: Sounds good, I'll start switching over that component.

**Comment by maxdiebold-erg** [comment]
@cschwinderg Good call on the `@reach/dialog`, that really simplified the modal component. Since we're only using the one modal, I went ahead and combined the `Modal` and `DownloadModal` components into one.

**Review by cschwinderg — APPROVED** [review]
Looks good! Thank you for making the switch to `@reach/dialog`.

**Pull request merged**